{"name": "User can add a product to cart", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[./span[contains(text(), 'subtotal')]]\"}\n  (Session info: chrome=138.0.7204.169); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x000000010143b570 cxxbridge1$str$ptr + 2731064\n1   chromedriver                        0x0000000101433468 cxxbridge1$str$ptr + 2698032\n2   chromedriver                        0x0000000100f823f8 cxxbridge1$string$len + 90664\n3   chromedriver                        0x0000000100fc971c cxxbridge1$string$len + 382284\n4   chromedriver                        0x000000010100ab1c cxxbridge1$string$len + 649548\n5   chromedriver                        0x0000000100fbda0c cxxbridge1$string$len + 333884\n6   chromedriver                        0x00000001013fe5f4 cxxbridge1$str$ptr + 2481340\n7   chromedriver                        0x000000010140185c cxxbridge1$str$ptr + 2494244\n8   chromedriver                        0x00000001013df248 cxxbridge1$str$ptr + 2353424\n9   chromedriver                        0x0000000101402118 cxxbridge1$str$ptr + 2496480\n10  chromedriver                        0x00000001013d02f8 cxxbridge1$str$ptr + 2292160\n11  chromedriver                        0x000000010142200c cxxbridge1$str$ptr + 2627284\n12  chromedriver                        0x0000000101422198 cxxbridge1$str$ptr + 2627680\n13  chromedriver                        0x00000001014330a4 cxxbridge1$str$ptr + 2697068\n14  libsystem_pthread.dylib             0x0000000190156c0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x0000000190151b80 thread_start + 8\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/cart_steps.py\", line 26, in verify_cart_items\n    cart_summary = context.driver.find_element(*CART_SUMMARY).text\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 748, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Open target main page", "status": "passed", "start": 1753566613234, "stop": 1753566615093}, {"name": "When Search for chair", "status": "passed", "start": 1753566615093, "stop": 1753566615345}, {"name": "And Click on Add to Cart button", "status": "passed", "start": 1753566615346, "stop": 1753566615800}, {"name": "And Store product name", "status": "passed", "start": 1753566615800, "stop": 1753566617703}, {"name": "And Confirm Add to Cart button from side navigation", "status": "passed", "start": 1753566617704, "stop": 1753566617762}, {"name": "And Open cart page", "status": "passed", "start": 1753566617762, "stop": 1753566618528}, {"name": "Then Verify cart has 1 item(s)", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[./span[contains(text(), 'subtotal')]]\"}\n  (Session info: chrome=138.0.7204.169); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x000000010143b570 cxxbridge1$str$ptr + 2731064\n1   chromedriver                        0x0000000101433468 cxxbridge1$str$ptr + 2698032\n2   chromedriver                        0x0000000100f823f8 cxxbridge1$string$len + 90664\n3   chromedriver                        0x0000000100fc971c cxxbridge1$string$len + 382284\n4   chromedriver                        0x000000010100ab1c cxxbridge1$string$len + 649548\n5   chromedriver                        0x0000000100fbda0c cxxbridge1$string$len + 333884\n6   chromedriver                        0x00000001013fe5f4 cxxbridge1$str$ptr + 2481340\n7   chromedriver                        0x000000010140185c cxxbridge1$str$ptr + 2494244\n8   chromedriver                        0x00000001013df248 cxxbridge1$str$ptr + 2353424\n9   chromedriver                        0x0000000101402118 cxxbridge1$str$ptr + 2496480\n10  chromedriver                        0x00000001013d02f8 cxxbridge1$str$ptr + 2292160\n11  chromedriver                        0x000000010142200c cxxbridge1$str$ptr + 2627284\n12  chromedriver                        0x0000000101422198 cxxbridge1$str$ptr + 2627680\n13  chromedriver                        0x00000001014330a4 cxxbridge1$str$ptr + 2697068\n14  libsystem_pthread.dylib             0x0000000190156c0c _pthread_start + 136\n15  libsystem_pthread.dylib             0x0000000190151b80 thread_start + 8\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/cart_steps.py\", line 26, in verify_cart_items\n    cart_summary = context.driver.find_element(*CART_SUMMARY).text\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 748, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1753566618528, "stop": 1753566622659}, {"name": "Then Verify cart has correct product", "status": "skipped", "start": 1753566622793, "stop": 1753566622793}], "start": 1753566611913, "stop": 1753566622793, "uuid": "797bb29b-c4ee-4f2d-b3b9-7885820d5a11", "historyId": "ab6f8215bcf4af1938cb6fa85afdc0ca", "fullName": "Target search test cases: User can add a product to cart", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "Target search test cases"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}